imap_processing/cdf/config/imap_ialirt_global_cdf_attrs.yaml

```
@@ -0,0 +1,18 @@
+instrument_base: &instrument_base
+  Descriptor: IALiRT>IMAP IALiRT (IMAP Active Link in Real Time)
+  TEXT: >
+    The Interstellar Mapping and Acceleration Probe (IMAP) mission includes the Active Link
+    for Real-Time (I-ALiRT) system, which provides continuous, low-latency space weather data.
+    I-ALiRT broadcasts real-time measurements from IMAP via NASA’s Deep Space Network and
+    international ground stations. It has data from 5 of the IMAP instruments:
+    SWE (Solar Wind Electron instrument), CoDICE (Compact Dual Ion Composition Experiment),
+    HIT (High-energy Ion Telescope), MAG (Magnetometer),
+    and SWAPI (Solar Wind and Pickup Ion instrument).
```

For consistency I'd remove the extra "instrument" from SWE and SWAPI.
```suggestion
    SWE (Solar Wind Electron), CoDICE (Compact Dual Ion Composition Experiment),
    HIT (High-energy Ion Telescope), MAG (Magnetometer),
    and SWAPI (Solar Wind and Pickup Ion).
```